### DISCUSSION TITLE
Rearrange/Sort Entry Lines in a Journal Entry

### DISCUSSION THREAD

**Discussion by Chibimonster** [burkybang/SuiteAdvanced-Feedback#147] [Ideas]
I didn't see anything related to this, so I apologize for missing it. 

Is it possible to sort the lines or drag/rearrange the entry line order when editing a journal entry? That feature is currently available when editing an advanced intercompany journal but is not in the standard journal entry form for some reason.

**Comment by burkybang** [comment]
I'll keep it in mind, but I don't think it would be possible for me to implement that (or at least it would be incredible difficult).

**Reply by Chibimonster** [reply]
Thank you so much!!!
